pkg/helper/otel/otel_config.go

```diff
@@ -0,0 +1,135 @@
+package otel
+
+import (
+	_ "embed"
+	"encoding/json"
+	"sort"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+)
+
+// openTelemetryDefaultTransformRules defined the default Open Telemetry format
+// See https://github.com/rhobs/observability-data-model/blob/main/network-observability.md#format-proposal
+
+//go:embed otel-config.json
+var rawOtelConfig []byte
+var otelConfig *map[string]string
+var otelRules *[]api.GenericTransformRule
```

Comment:
same here for slice & map, pointer is unnecessary

Reply:
done in https://github.com/netobserv/network-observability-operator/pull/809/commits/5eecbd5f758b2da4a2b422b57567de0a5fa0a8af